src/rgw/rgw_pubsub.cc

```diff
@@ -337,6 +337,7 @@ void rgw_pubsub_topic::dump(Formatter *f) const
   encode_json("dest", dest, f);
   encode_json("arn", arn, f);
   encode_json("opaqueData", opaque_data, f);
+  encode_json("Policy", policy_text, f);
```

Comment:
i think that JSON should be `camelCase`, so we should change to "policy"

Reply:
Done.